UKHomeOffice/cerberus-service
# COP-11460 - Movement details

## Description
This PR adds the `arrival` & `departure` nodes to the `journey` node within the submission payload when issuing a target.

## To Test
1) Pull & run against dev.
2) Claim and task and click on `Issue target` and fill out the form then send it.
3) Import the postman collection located [here](https://gitlab.digital.homeoffice.gov.uk/cop/cop-targeting-api/-/blob/master/postman/cop-targeting-api.postman_collection.json) and and the environment file located [here](https://gitlab.digital.homeoffice.gov.uk/cop/cop-targeting-api/-/blob/master/postman/cop-targeting-api-dev.postman_environment.json) if not already imported.
4) Within postman, locate the folder called `cop-targeting-api`, go through the process of authentication.
5) Once this is done, within the folder `cop-targeting-api` locate the sub-folder `target-management` and select the `GET` request `target-by-id`.
6) Within the environment window, change the global variable of `{{target-id}}` to that of the task that was issued in the **`Step 2`**.
7) Inspect the response within postman's response panel and verify that the response contains the `journey` node (you can search for it also using the search functionality). Within the journey node, verify that the`arrival` &`departure` nodes are present.

### Supplemental
- Some changes has also been made in regards to the photograph nodes to fix an issue in another separate PR where each file input within a container were not being assigned unique ids.
